test/inductor/test_cpp_wrapper.py

```diff
@@ -112,6 +114,7 @@ def fn(self):
             tests.tearDownClass()
 
     fn.__name__ = test_name
+    fn.__dict__ = copy.deepcopy(func.__dict__)
```

Comment:
I'm a bit confused. Isn't `fn` a local function created just above? It's dict attr is not shared with anyone else and is fresh from creation?

Reply:
The point is the original function may have had tags on it (e.g., expectedFailure is implemented this way), and when we make a new "copy" of the function that we created (as you say), we should propagate those tags.

Reply:
Ho sure, my bad, misread this...